docs/source/project_management/ci_maintainer_notes.rst

```diff
@@ -220,3 +234,21 @@ The ``test_docs.yaml`` file will call the ``deploy_docs_tmpl.yaml`` file of the
 You do not need to change anything else.
 
 This ``format.yaml`` file will call the ``format_tmpl.yaml`` file of the centralized ``.github`` repo and take care of the rest of the tasks for you!!!
+
+Automatically Adding License Headers
+====================================
+
+NWChemEx uses `Apache License (2.0) <https://www.apache.org/licenses/LICENSE-2.0>`__. We want all our source codes (except for some configuration, test and input files) to have the proper license header when deployed. We have workflow files for automatically adding the license header. The main workflow file resides in ``.github/.github/workflows/add_licenses_master.yaml`` which is in the ``.github`` repo. You may recall that it is where all template workflows live. You do not need to do any change in this file. You will need to add two workflow files to get it running for your new repo. 
+
+1. Firstly, you can copy the ``add_licenses.yaml`` file from any other repo ( except ``.github`` ) listed in the very beginning of this documentation and paste it in your new repository's ``.github/workflows`` path which will be something like ``new-repo-name/.github/workflows/``. For example, you can copy the ``add_licenses.yaml`` file from ``NWChemEx`` repository which resides in the path ``NWChemEx/.github/workflows/add_licenses.yaml`` of the repository. This workflow file calls the ``add_licenses_master.yaml`` wrokflow file which resides in the ``.github`` repo with two parameters: ``config_file`` and ``token``. The value for ``token`` should be ``{{ secrets.GITHUB_TOKEN }}``, which is same for every repository.
```

Comment:
```suggestion
1. Firstly, you can copy the ``add_licenses.yaml`` file from any other repo ( except ``.github`` ) listed in the very beginning of this documentation and paste it in your new repository's ``.github/workflows`` path which will be something like ``new-repo-name/.github/workflows/``. For example, you can copy the ``add_licenses.yaml`` file from ``NWChemEx`` repository which resides in the path ``NWChemEx/.github/workflows/add_licenses.yaml`` of the repository. This workflow file calls the ``add_licenses_master.yaml`` workflow file which resides in the ``.github`` repo with two parameters: ``config_file`` and ``token``. The value for ``token`` should be ``{{ secrets.GITHUB_TOKEN }}``, which is same for every repository.
```

Reply:
@hjjvandam and @twindus if you select a line (or lines) of code and then click the page icon with the $\pm$ icon you can suggest the changes yourself.

Reply:
OK, good to know.

Reply:
I guess I knew that, but forgot!  :)